Sources/MapboxNavigation/Resources/Assets.xcassets/recenter.imageset/Contents.json

```diff
@@ -1,12 +1,21 @@
 {
   "images" : [
     {
+      "filename" : "recenter.pdf",
```

Comment:
This replaces originally used recenter image, I did a quick search and couldn't find its usage. Let me know if I'm mistaken by any chance.

Reply:
I don’t think we’ve used the up-arrow image for a long time, if ever.